evm-validate/src/lib.rs

```diff
@@ -417,3 +468,764 @@ fn validate_transaction_hashes(
 
     Ok(())
 }
+
+// AccessListWrapper is a wrapper around the AccessList type that allows it to be deserialized from a byte array.
+#[derive(Deserialize)]
+pub struct AccessListWrapper(pub Vec<AccessListItemWrapper>);
+
+impl Into<AccessList> for AccessListWrapper {
+    fn into(self) -> AccessList {
+        AccessList(self.0.into_iter().map(|item| item.into()).collect())
+    }
+}
+
+// AccessListItemWrapper is a wrapper around the AccessListItem type that allows it to be deserialized from a byte array.
+#[derive(Deserialize)]
```

Comment:
don't think deserialize is needed. Should be able to iterate input directly and construct these objects. Can even directly construct the real alloy types instead of having typedefs here

Reply:
The alloy_eips::eip2930::{AccessList, AccessListItem} types don't implement deserialize, so I needed a wrapper around it.